test/es6FromFHIRJSONTest.js

```diff
@@ -0,0 +1,73 @@
+const { expect } = require('chai');
+const setup = require('./setup');
+const { TestContext, importResult } = require('./test_utils');
+require('babel-register')({
+  presets: [ 'es2015' ]
+});
+
+setup('./test/fixtures/spec', './build/test', true);
+const context = new TestContext();
+context.setupAjvJson('./build/test/schema');
+context.setupAjvFhir('./test/fixtures/fhir-schema', 'FHIR_STU_3');
+
+describe('#FromFHIR', () => {
+  describe('#PatientDirectMapEntry()', () => {
+    const PatientDirectMapEntry = importResult('shr/fhir/PatientDirectMapEntry');
+    it('should deserialize a FHIR JSON instance', () => {
+      const json = context.getFHIR('PatientDirectMapEntry');
+      const entry = PatientDirectMapEntry.fromFHIR(json);
+      expect(entry).instanceOf(PatientDirectMapEntry);
+
+      const roundTrip = entry.toFHIR();
+      expect(roundTrip).to.eql(json);
```

Comment:
So a round-trip test is a good idea, but if there are bugs in `toFHIR` (and I suspect there are), then round-tripping only ensures that you have corresponding bugs in `fromFHIR`!  I'd like to see at least _some_ tests that actually inspect the result to ensure that all of the data is where it should be in the class instance.